x/cert/internal/keeper/certificate.go

```diff
@@ -47,18 +49,21 @@ func (k Keeper) GetCertificateByID(ctx sdk.Context, id types.CertificateID) (typ
 }
 
 // GetNewCertificateID gets an unused certificate ID for a new certificate.
-func (k Keeper) GetNewCertificateID(ctx sdk.Context, certifier sdk.AccAddress, certType types.CertificateType,
+func (k Keeper) GetNewCertificateID(ctx sdk.Context, certType types.CertificateType,
 	certContent types.RequestContent) (types.CertificateID, error) {
```

Comment:
The variable name `certContent` is confusing. Maybe it would be better to name it `reqContent` or `content`?